src/sentry/api/serializers/models/sentry_app.py

```diff
@@ -30,15 +30,7 @@ def serialize(self, obj, attrs, user):
         if is_active_superuser(env.request) or (
             hasattr(user, 'get_orgs') and obj.owner in user.get_orgs()
         ):
-            if obj.is_internal:
-                install = obj.installations.first()
-                data.update({
-                    'installation': {
-                        'uuid': install.uuid,
-                    },
-                    'token': install.api_token.token,
-                })
-            else:
+            if not obj.is_internal:
```

Comment:
@MeredithAnya Don't we need `installation` and `token` from before? Or did they move somewhere else?

Reply:
we won't render the installation id anymore and then the tokens were being grabbed from the `/api-tokens` route